### ISSUE TITLE
Media JSON object is null

### ISSUE THREAD

**Issue by rocapal** [mwsrp/whos-that-whaleshark-android#1]
Using the API to get the shark encounters I have found that 'media' JSON object (which is a JSON array), has assigned a Null value. I guess it could be better that for those cases 'media' object gets a value of empty array []. I think even it's going to be easier for the JSON parsers.

**Comment by yogaraptor** [comment]
Agreed! When we redevelop the api over at [mwsrp/critter-api](https://github.com/mwsrp/critter-api) we'll definitely implement that
